scripts/deploy.py

```diff
@@ -0,0 +1,296 @@
+from brownie.network import accounts

```

Comment:
Since this script is used to deploy the ayii product we should rename is `deploy_ayii.py`

Reply:
done